mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala

```diff
@@ -549,6 +555,106 @@ class VizServlet extends ScalatraServlet {
     }
   }
 
+  post("/ga4gh/variants/search") {
+    val jsonPostString = request.body
+    val searchVariantsRequest: SearchVariantsRequestGA4GH = net.liftweb.json.parse(jsonPostString)
+      .extract[SearchVariantsRequestGA4GH]
+
+    if (!VizReads.materializer.variantContextExist)
+      VizReads.errors.notFound
+    else {
+      val viewRegion = ReferenceRegion(searchVariantsRequest.referenceName,
+        searchVariantsRequest.start.toLong,
+        VizUtils.getEnd(searchVariantsRequest.end.toLong,
+          VizReads.globalDict(searchVariantsRequest.referenceName)))
+
+      //currently only supporting a single call set id in the query, while API supports a list of call set ids
+      val key = searchVariantsRequest.callSetIds.head
+      contentType = "json"
+
+      val dictOpt = VizReads.globalDict(viewRegion.referenceName)
+
+      if (dictOpt.isDefined) {
+        var results: Option[String] = None
+
+        val binning: Int = 1
```

Comment:
that's fine, though we then need to hard-code "1" for bin parameter for some of the functions calls below that need bin parameter